frontend/src/components/navbar.css

```diff
@@ -0,0 +1,46 @@
+nav {
```

Comment:
can you put a `.` before each of the CSS class names in the file and refactor the filename to be `NavBar.module.css`